content/blog/2020-06-03-how-learnr-saved-the-day/index.markdown

```diff
@@ -0,0 +1,126 @@
+---
+title: How learnr Saved the Day
+author: Ezgi Karaesmen
+date: '2020-06-03'
+description: |
+   My attempt at briefly introducing R to a group of teenagers and how learnr saved me from boring students to death.
+slug: learnr-saved-the-day
+photo:
+  url: https://unsplash.com/photos/6yjAC0-OwkA
+  author: Esteban Lopez
+categories:
+  - teach
+tags:
+  - learnr
+---
+
+Instructors and learners alike have had to quickly adapt to virtual education due to the ongoing pandemic. The [learnr](https://rstudio.github.io/learnr/) package has become a popular tool to ease that transition (see Allison Horst's great  [post](https://education.rstudio.com/blog/2020/05/learnr-for-remote/)).
+
+Before our lives was completely changed by the pandemic, however; I was fortunate enough to conduct a *real life* introductory R session with a group of **teenagers**. I was able to create the materials for this session, [Understanding Injury Data with R](https://suchestoncampbell-lab.shinyapps.io/SummerCamp2019/), with the learnr package and the source code is available on [GitHub](https://github.com/karaesmen/SummerCamp2019/tree/master/tutorial). Although in person examples of teaching may not be the most relevant topic under these circumstances, I thought that sharing my experience of teaching a younger audience using learnr could offer a unique perspective. 
```

Comment:
1. Reviewing here because the lines wrap (they don't in the `.Rmarkdown`.
2. Italics and bold in one sentence seems a bit much - I'd remove both.
3. I use "in person" instead of "real life".